src/main.cpp

```diff
@@ -2,7 +2,7 @@
 
 int main(int argc, char **argv)
 {
-    auto test = new std::string;
+    auto test = new std::string;  // :)
```

Comment:
warning: 'auto test' can be declared as 'auto *test' [readability-qualified-auto]

```suggestion
    auto *test = new std::string;  // :)
```